timesync/clock_test.go

```diff
@@ -105,9 +105,12 @@ func TestTicker_TickFutureGenesis(t *testing.T) {
 
 func TestTicker_TickPastGenesis(t *testing.T) {
 	tmr := &RealClock{}
-	ticker := NewTicker(tmr, 2*time.Second, tmr.Now().Add(-4*time.Second))
+	ticker := NewTicker(tmr, 1*time.Second, tmr.Now().Add(-3900*time.Millisecond))
 	sub := ticker.Subscribe()
 	ticker.Start()
+	start := time.Now()
 	x := <-sub
-	assert.Equal(t, types.LayerID(3), x)
+	duration := time.Since(start)
+	assert.Equal(t, types.LayerID(4), x)
```

Comment:
should'nt it be tick 5 ?. 
tick 1 is in genesis time, then we have 3 more ticks. (2,3,4). so the next tick should be 5.

Reply:
fixed